### DISCUSSION TITLE
👏  How to contribute

### DISCUSSION THREAD

**Discussion by jw-foss** [element-plus/element-plus#3326] [Documentation]
## Welcome to ElementPlus

To be able to participate, you will need to:
1. Fork this repo
2. Check the project board [Migration Backlog](https://github.com/element-plus/element-plus/projects/1) and find the component you are going to work on.
3. Make sure that you read README before coding.
4. After you complete your coding, Pull Request to this repo and loop some reviewers into

**Comment by luxueyan** [comment]
I have no permission to convert to a card to an issue. How operate it.

**Comment by jw-foss** [comment]
> I have no permission to convert to a card to an issue. How operate it.

I'll help you with that, but before you doing so, please take a look on the feature itself

**Comment by g3r4n** [comment]
All project tasks are "In progress", Is there any area where we can help ?

**Comment by zazzaz** [comment]
@g3r4n code review is welcomed.

**Comment by Pmj136** [comment]
I have found that the 'value' property of many components does not work and requires the 'modelValue' property to be used. 'v-mode' is not affected. Whereas the documentation says 'value' is available. Do you need me to help update the documents or components?

**Comment by jw-foss** [comment]
@Pmj136  Yes! We do, if you are interested in helping us, PRs are welcomed.

**Comment by JinxCarry9527** [comment]
How do I know which issues are to be solved

**Comment by iamkun** [comment]
@JinxCarry9527 the issue that is not closed.
